pkg/app/piped/trigger/BUILD.bazel

```diff
@@ -18,7 +18,6 @@ go_library(
         "//pkg/filematcher:go_default_library",
         "//pkg/git:go_default_library",
         "//pkg/model:go_default_library",
-        "//pkg/version:go_default_library",
```

Comment:
nits, not sure why does this related to this change

Reply:
No. A new file was added and I ran `make gazelle`. It seems that the version package is no longer needed for that trigger package.